ygot/render_test.go

```diff
@@ -1312,14 +1314,23 @@ func notificationSetEqual(a, b []*gnmipb.Notification) bool {
 
 	matchall := true
 	for _, aelem := range a {
-		var matched bool
+		var updateMatched, deleteMatched bool
 		for _, belem := range b {
+			if aelem == nil && belem != nil || belem == nil && aelem != nil || aelem == nil && belem == nil {
```

Comment:
(pre-existing, optional) Names here and below should be aElem, bElem...

Reply:
Fixed.